build/codegen.sh

```diff
@@ -24,7 +24,7 @@ source build/common.sh
 
 export PYTHONPATH='.:vendor/'
 
-if test -z "${IN_NIX_SHELL:-}"; then
+if test -z "${IN_NIX_SHELL:-}${NIX_BUILD_TOP:-}"; then
```

Comment:
I don't quite understand why, but this broke nix-build on my local macOS and not in CI (suspect a clean/dirty work dir thing).